glob.go

```diff
@@ -98,6 +100,39 @@ func doGlob(fsys fs.FS, pattern string, m []string, firstSegment bool) (matches
 	return
 }
 
+// FilepathGlob returns the names of all files matching pattern or nil if there is
+// no matching file.
+// The syntax of pattern is the same as in Match(). The pattern
+// may describe hierarchical names such as usr/*/bin/ed.
+//
+// FilepathGlob ignores file system errors such as I/O errors reading directories.
+// The only possible returned error is ErrBadPattern, reporting that the
+// pattern is malformed.
+//
+// Note: this is meant as a drop-in replacement for path/filepath.Glob().
+// The matching paths have the same format as what is returned by path/ilepath.Glob().
+func FilepathGlob(pattern string) ([]string, error) {
+	pattern = filepath.Clean(pattern)
+	base, f := SplitPattern(pattern)
+	if base == "" {
+		base = string(os.PathSeparator)
+	}
+	fs := os.DirFS(base)
+	matches, err := Glob(fs, f)
+	if err != nil {
+		return nil, err
+	}
+	for i := range matches {
+		matches[i] = filepath.Join(base, matches[i])
+	}
+	if len(matches) == 0 {
+		if _, err := os.Stat(pattern); err == nil {
```

Comment:
Would this ever happen? If `Glob(pattern)` returns no results, but `Stat(pattern)` does, that sounds like a bug in Glob.